### PR TITLE
Remove the old plcontainer API

### PR DESCRIPTION
**Pull request by zhangh43**
Does not support old API 'plcontainer configure' and
'plcontainer install'

### CODE REVIEW

File: management/bin/plcontainer [hunk 1]
```diff
@@ -44,24 +44,19 @@ def parseargs():
...
-                        " -l plcontainer_language_type")
-    install_parser = subparsers.add_parser('install', help='Install a dokcer image and add it to configuration file')
-
     image_add_parser = subparsers.add_parser("image-add", help = "Install specified docker image on all hosts")
     image_delete_parser = subparsers.add_parser("image-delete", help = "Delete specified docker image on all hosts")
     image_list_parser = subparsers.add_parser("image-list", help = "Show existing container images on the current host")
 
     runtime_add_parser = subparsers.add_parser("runtime-add", help = "Add a container runtime configuration on all hosts")
-    runtime_replace_parser = subparsers.add_parser("runtime-replace", help = "Replace/Add with a container runtime configuration on all hosts")
-    runtime_show_parser = subparsers.add_parser("runtime-show", help = "Show all container runtime configurations. Use -r/--runtime to show specific runtime")
```
Comment (paul-guo-) [inline management/bin/plcontainer]: runtime-show supports showing for a specified runtime id. Maybe

"Show configurations for one or all runtimes."?

File: management/bin/plcontainer [hunk 2]
```diff
@@ -44,24 +44,19 @@ def parseargs():
...
-    runtime_delete_parser = subparsers.add_parser("runtime-delete", help = "Delete a container runtime configuration on all hosts")
```
Comment (paul-guo-) [inline management/bin/plcontainer]: Also this just deletes for one only.

File: management/bin/plcontainer [hunk 3]
```diff
@@ -44,24 +44,19 @@ def parseargs():
...
-    runtime_edit_parser = subparsers.add_parser("runtime-edit", help = "editor to edit and then update configuration on all hosts")
-    runtime_backup_parser = subparsers.add_parser("runtime-backup", help = "Backup the local runtime configuration file")
-    runtime_restore_parser = subparsers.add_parser("runtime-restore", help = "Restore the runtime configuration from a file and distribute them to all hosts")
+    runtime_replace_parser = subparsers.add_parser("runtime-replace", help = "Replace with a container runtime configuration on all hosts")
+    runtime_show_parser = subparsers.add_parser("runtime-show", help = "Show container runtime configurations on the current host")
+    runtime_delete_parser = subparsers.add_parser("runtime-delete", help = "Delete container runtime configurations on all hosts")
+    runtime_edit_parser = subparsers.add_parser("runtime-edit", help = "Using editor to edit and then update configurations on all hosts")
```
Comment (paul-guo-) [inline management/bin/plcontainer]: Use?

**Comment by paul-guo-** [comment]
Please also remove get_last_backup

File: management/bin/plcontainer (hunk 1 above)
Comment (zhangh43) [reply management/bin/plcontainer]: specific runtime id is an option of runtime-show.
I prefer to show it by running 'plcontainer runtime-show -h'

File: management/bin/plcontainer (hunk 2 above)
Comment (zhangh43) [reply management/bin/plcontainer]: fixed

**Pull request merged**